src/main/java/picard/util/IntervalListTools.java

```diff
@@ -70,7 +96,7 @@
  *       I=input_2.interval_list \\
```

Comment:
This code could avoid quoting like this by wrapping the code in `{@code}`. e.g.
```
 * <pre>
 * {@code
 * java -jar picard.jar IntervalListTools \
 *       ACTION=CONCAT \
 *       I=input.interval_list \
 *       I=input_2.interval_list \
 *       O=new.interval_list
 * }
 * </pre>
```
This is just a good trick to remember in general for code blocks, not really required here though.

Reply:
nice. we should show this to the comms team...

Reply:
So use `{@code ... }` and remove the extra`\` per line?

Reply:
Using `@code` has other benefits as it also allows `<` `>` and `&` in javadoc without quoting.